src/Kernels/Plasticity.cpp

```diff
@@ -93,35 +97,44 @@ namespace seissol::kernels {
     m2nKrnl.initialLoading = plasticityData->initialLoading;
     m2nKrnl.execute();
 
+    // Computes m = s_{ii} / 3.0 for every node
     kernel::plComputeMean cmKrnl;
     cmKrnl.meanStress = meanStress;
     cmKrnl.QStressNodal = QStressNodal;
     cmKrnl.selectBulkAverage = init::selectBulkAverage::Values;
     cmKrnl.execute();
 
+    /* Compute s_{ij} := s_{ij} - m delta_{ij},
+     * where delta_{ij} = 1 if i == j else 0.
+     * Thus, s_{ij} contains the deviatoric stresses. */
     kernel::plSubtractMean smKrnl;
     smKrnl.meanStress = meanStress;
     smKrnl.QStressNodal = QStressNodal;
     smKrnl.selectBulkNegative = init::selectBulkNegative::Values;
     smKrnl.execute();
 
+    // Compute I_2 = 0.5 s_{ij} s_ji for every node
     kernel::plComputeSecondInvariant siKrnl;
     siKrnl.secondInvariant = secondInvariant;
     siKrnl.QStressNodal = QStressNodal;
     siKrnl.weightSecondInvariant = init::weightSecondInvariant::Values;
     siKrnl.execute();
 
+    // tau := sqrt(I_2) for every node
     for (unsigned ip = 0; ip < tensor::secondInvariant::size(); ++ip) {
       tau[ip] = sqrt(secondInvariant[ip]);
     }
 
+    // Compute tau_c for every node
     for (unsigned ip = 0; ip < tensor::meanStress::size(); ++ip) {
       taulim[ip] = std::max((real) 0.0, plasticityData->cohesionTimesCosAngularFriction -
                                         meanStress[ip] * plasticityData->sinAngularFriction);
     }
 
     bool adjust = false;
     for (unsigned ip = 0; ip < tensor::yieldFactor::size(); ++ip) {
+      // Compute yield := (t_c / tau - 1) r for every node,
+      // where r = 1 - exp(-timeStepWidth / T_v)
```

Comment:
what you call r is relaxTime at L140?
but if r = 1 - exp(-timeStepWidth / T_v), it is not a time, right?

Reply:
Yes I think it not a time, this variable should be named differently. e.g. one_minus_exp_dt_Tv
But I now understand that what you send me is equivalent to the changes in the PR if dt/Tv
because 1-exp(-dt/Tv)~ dt/Tv